internal/job/restore/restore.go

```diff
@@ -147,6 +148,9 @@ func (r *Restore) loopCopyChunk(privateData *restorePrivateData, rawImageSize ui
 		rawImagePath := r.nodeLocalVolumeRepo.GetRawImagePath()
 		restoreVolPath := r.restoreVol.GetPath()
 		if err := r.restoreVol.CopyChunk(rawImagePath, i, r.rawImageChunkSize); err != nil {
+			if errors.Is(err, csumio.ErrChecksumMismatch) {
+				return fmt.Errorf("%w: raw image chunk %d", csumio.ErrChecksumMismatch, i)
+			}
```

Comment:
Let's avoid discarding `err`.


```suggestion
			if errors.Is(err, csumio.ErrChecksumMismatch) {
				return fmt.Errorf("raw image chunk %d: %w", i, err)
			}
```